services/fdb/client/client.go

```diff
@@ -3461,3 +3566,193 @@ func (p *fdbServerVersionCmd) Execute(ctx context.Context, f *flag.FlagSet, args
 
 	return retCode
 }
+
+const fdbMoveDataCLIPackage = "fdbmovedata"
+
+func GetSubpackage(f *flag.FlagSet) *subcommands.Commander {
+	c := client.SetupSubpackage(fdbMoveDataCLIPackage, f)
+	c.Register(&fdbMoveDataCopyCmd{}, "")
+	c.Register(&fdbMoveDataWaitCmd{}, "")
+	return c
+}
+
+// This context detachment is temporary until we use go1.21 and context.WithoutCancel is available.
```

Comment:
I think it's a good idea to move this struct closer to where it's being used, which is right before `func (r *fdbMoveDataWaitCmd) Execute`